src/main/java/Lottery.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    public static Ball getRandomBall() {
+        Colors color = ColorsSupplier.getRandomColor();
+        int number = RANDOM.nextInt(100) + 1; // generate a number from 1 to 100

```

Comment:
magic numbers should be constants with proper naming, with such approach you won't need to comment your code, it will be self-explanatory